src/main/java/org/jenkinsci/plugins/ghprb/extensions/comments/GhprbCommentFile.java

```diff
@@ -42,9 +51,50 @@ public String postBuildComment(Run<?, ?> build, TaskListener listener) {
 
             try {
                 String content = null;
-                if (build instanceof Build<?, ?>) {
-                    final FilePath workspace = ((Build<?, ?>) build).getWorkspace();
-                    final FilePath path = workspace.child(scriptFilePathResolved);
+                FilePath workspace;
+                FilePath path;
```

Comment:
looks like these don't need to be declared in the outerscope.  it's used within each if block anyway.  workspace isn't even used in your if block.